config/default.json

```diff
@@ -9,6 +9,8 @@
   },
   "rabbitmq": {
     "host": "localhost",
-    "port": 5672
+    "port": 5672,
+    "exchange": "harvests",
+    "topic": "conceptHarvestTrigger"
```

Comment:
som var bestemt, hver meldingtyp får sit eget topic, så du kan ikke ha en topic. Hva du kan ha, er topicSuffix=HarvestTrigger. Men også det ikke veldig lurt, fårdi det endrer seg ikke i miljøene. så helst bevege det til modul som const

Reply:
topic i rabbitmq er veldig fleksibel. den støtter . notasjon. Så vi kan ha concept.harvestTrigger eller all.concept.harvestTrigger eller hvordan vi absolutt vil. Det er bare å gi meg en hel liste, så kan jeg fikset det mtp skalerbarhet

Reply:
det står jo i speccen som en eksempel av en meldingstyp,  men senere blir det 4 eller mer. og hver melding for sitt eget topic

Reply:
Men vi skal bare sende 1 av dem fra fdk-harvest-admin